### PR TITLE
Feature g9 map object edition

### PR DESCRIPTION
**Pull request by kmilena98**
Adding possibility to edit map object

### CODE REVIEW

File: HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs
Comment (vlaksi) [inline HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs:78]: Give a more descriptive name for 'e'  or do this:

```sh
    handler(this,  new PropertyChangedEventArgs(propertyName));
```

File: HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs
Comment (vlaksi) [inline HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs:93]: Why is dead code here?  Try to avoid dead code.

File: HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs
Comment (vlaksi) [inline HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs]: What is this doing?

**Review by vlaksi — CHANGES_REQUESTED** [review]

**Comment by milana-todorovic** [comment]
This branch was separated from a feature branch instead of develop, so it can't really be merged to develop like this according to our workflow. One possible solution would be to delete this branch and pull request, make a new branch from develop once feature-g8-display-all-information is merged, and then copy the changes you made here to that new branch and open a new pull request. Another option would be to not merge the feature-g8-display-all-information branch and merge this instead, but I'd rather not do that since commits up to the last one have already been reviewed on the other pull request. The third option is to close this pull request, merge this branch into feature-g8-display-all-information, and have this last commit reviewed on that pull request as well. Whichever of those works for you is fine.

File: HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs
Comment (kmilena98) [reply HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs]: i didnt finished it yet

**Comment by VasilijeBursac** [comment]
[(3) g10: Change additional information (description, working hours,...)](https://trello.com/c/VHHUGlJb/10-3-g10-change-additional-information-description-working-hours)

**Comment by VasilijeBursac** [comment]
[(3) g9: Change basic information (name)](https://trello.com/c/omjRBFsz/9-3-g9-change-basic-information-name)

**Comment by kmilena98** [comment]
[(3) g9: Change basic information (name)](https://trello.com/c/omjRBFsz/9-3-g9-change-basic-information-name)

**Comment by kmilena98** [comment]
[(3) g9: Change basic information (name)](https://trello.com/c/omjRBFsz/9-3-g9-change-basic-information-name)

**Comment by vlaksi** [comment]
@VasilijeBursac @kmilena98 what are you decide with this branch? Milana gives you a proposal above.

**Comment by VasilijeBursac** [comment]
> @VasilijeBursac @kmilena98 what are you decide with this branch? Milana gives you a proposal above.

@vlaksi Milena will solve that with branches. She can give you more details about this branch because she worked on it.

**Comment by milana-todorovic** [comment]
Closed without merge because of workflow issues. Changes moved to branch feature-g9-edition.

**Pull request closed without merging**
